language/libra-vm/src/counters.rs

```diff
@@ -69,3 +69,33 @@ pub static TXN_GAS_USAGE: Lazy<Histogram> = Lazy::new(|| {
     )
     .unwrap()
 });
+
+pub static VM_RUN_SCRIPT_PROLOGUE_SECONDS: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        // metric name
+        "vm_run_script_prologue_seconds",
```

Comment:
All of these new metrics should follow the naming convention and start with a "libra_vm" prefix. Also for consistency with the other metrics in this file, you can drop the "metric name" and "metric description" comments -- I know some other source files list those comments every time, but it seems unnecessary to me and just makes the code harder to read.